sprocs/grading_jobs_insert.sql

```diff
@@ -40,15 +42,15 @@ BEGIN
     -- ######################################################################
     -- delegate the call
 
-    IF grading_method = 'Internal' THEN
+    IF grading_method_internal = True THEN
```

Comment:
Think this is going to be the choke point for adding another grading job (i.e. external/manual in parallel with internal). We may have to deal with ordering though... What if the internal job resolves, but the scores conflict with the external output? Do we have to do a split on the scores field as well?

Reply:
I agree that this is a choke point, but I am not sure how the internal job would resolve but then have a conflicting score with the external output if a question can only exclusively be external or internal.

Reply:
For now, it's not an issue since they are exclusive. In the future, it may be.

It's not an issue with adding a parallel manual workflow though, providing we tease that out.

Reply:
Yeah, agreed. I think the "parallel manual workflow" could be a call to a different sproc. It doesn't seem to make much sense that we have internal and external question types that are graded and then change their type to manual to be graded for manual grading, which then leads down a third possible delegated call here... 

Manual grading seems more like an action than a grading method type now.

Reply:
One additional note, we could use the weighted score in the question configuration to segment the final score and visualize what portion accounts for manual grading on the front-end. 

Don't know if this is too far down the road to think about, but it looks very close to achieving here.